modules/appeals_api/app/controllers/concerns/appeals_api/character_handling.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module CharacterHandling
+    extend ActiveSupport::Concern
+
+    OUTSIDE_WINDOWS_1252_PATTERN = /[^\u0000-\u0255]+/.freeze
+
+    included do
+      def validate_characters
+        characters = request_headers.to_s + params.to_s
+        characters.delete! '"'
+
+        render_invalid_characters_error if characters =~ OUTSIDE_WINDOWS_1252_PATTERN
+      end
+
+      def transliterate_name(str)
+        I18n.transliterate(str.to_s).gsub(%r{[^a-zA-Z\-\/\s]}, '').strip.first(50)
```

Comment:
Is this supposed to allow `/` characters? Because this regex won't actually allow that, due to escaping shenanigans. Found this out while working on the centralmail metadata bug (https://vajira.max.gov/browse/API-9471). I wasn't sure, so figured I'd just remove `/` for the bug, but it's something you should double check if you want to allow it.

Reply:
This is going away because your bug ticket accomplishes all the things.